src/coreclr/jit/llvm.cpp

```diff
@@ -1685,6 +1684,43 @@ Value* Llvm::localVar(GenTreeLclVar* lclVar)
     return llvmRef;
 }
 
+void Llvm::buildLocalField(GenTreeLclFld* lclFld)
+{
+    unsigned int lclNum = lclFld->GetLclNum();
+
+    Value*     structAddrValue;
+    LclVarDsc* varDsc = _compiler->lvaGetDesc(lclNum);
+
+    if (canStoreLocalOnLlvmStack(varDsc))
+    {
+        assert(m_allocas[lclNum] != nullptr);
+
+        structAddrValue = m_allocas[lclNum];
+    }
+    else
+    {
+        structAddrValue = getLocalVarAddress(lclFld);
```

Comment:
This should be handled in `ConvertShadowStackLocalNode`.

Once that's moved, you can delete `getLocalVarAddress` and `getLocalOffsetAtIndex`, they're unused.